docs/program/run.md

```diff
@@ -10,13 +10,26 @@ tags: ["client", "sdk", "application", "sdk", "fleet", "program"]
 
 After saving your [code sample](/program/#hello-world-the-code-sample-tab) and adding control logic with [Viam's SDKs](/program/apis/), run your program to control your Viam-connected robot.
 
-## Run Code Remotely
+### Authentication
+
+You must reference a robot's location secret to authenticate yourself to the robot.
```

Comment:
I don't think a user would know what they are supposed to do to "reference a robot's location secret", maybe include or add would be better?

Reply:
Oh I see now you were just movign this up... We can fix it separetely